src/main/java/com/github/jasminb/jsonapi/exceptions/UnregisteredClassForType.java

```diff
@@ -0,0 +1,18 @@
+package com.github.jasminb.jsonapi.exceptions;
+
+/**
```

Comment:
I suggest using better name for this, something like: `UnregisteredTypeException`.